File: python/player/utility.py
def print_with_buffer(string, buffer_length):
    buffer = ""
    for i in range(buffer_length):
        if i < len(string):
            buffer += string[i]
        else:
            buffer += " "
    return buffer


def string_is_valid(string):
    try:
        str(string)
        if len(string) == 0:
            return False

        return True
    except:
        return False


def print_field_error_message():
    print("Please try again, invalid input")

File: src/lib/nodejs/src/player/utility.js
module.exports.printWithBuffer = function (string, bufferLength) {
    let buffer = ""

    for (let i = 0; i < bufferLength; i++)
        if (i < string.length)
            buffer += string[i]
        else
            buffer += " "

    return buffer
}

module.exports.stringIsValid = function (string) {
    try {
        String(string)

        if (string.length === 0)
            return false

        return true
    } catch (e) {
        return false
    }
}

module.exports.printFieldErrorMessage = function () {
    console.log("Please try again, invalid input")
}